tests/command_line/test_find_bad_pixels.py

```diff
@@ -18,7 +18,7 @@ def test_find_bad_pixels(dials_data, tmp_path):
     count = 0
     for record in result.stdout.decode().split("\n"):
         if "mask" in record:
-            assert record.split()[-1] == "8"
+            assert record.split()[-1] == "16"
```

Comment:
We should probably test/verify the C-ordering vs Fortran-ordering... a little concerning that changing the ordering doesn't cause the test to fail. Either parse the output to build up the set of bad pixels to compare with known reference, or just test one or more expected row to assert that it is present in the output.

Reply:
Obviously json output would make this easier, although probably still useful to check the stdout.

Reply:
👍